python/lsst/ctrl/bps/parsl/sites/princeton.py

```diff
@@ -48,9 +48,9 @@ def get_executors(self) -> List[ParslExecutor]:
         Each executor should have a unique ``label``.
 
         The walltime default here is set so we get into the tiger-vshort QoS,
-        which will hopefully reduce the wait for us to get a node. Then, we have
-        one Slurm job running at a time (singleton) while another saves a spot
-        in line (max_blocks=2). We hope that this allow us to run almost
+        which will hopefully reduce the wait for us to get a node. Then, we
+        have one Slurm job running at a time (singleton) while another saves a
+        spot in line (max_blocks=2). We hope that this allow us to run almost
```

Comment:
Were the changes to the docstring necessary?

Reply:
L51 was [longer than 79 characters](https://developer.lsst.io/python/style.html?highlight=python+style+guide#docstring-and-comment-line-length-must-be-less-than-or-equal-to-79-columns), and so triggered my linter.